Applications/WebPortal/WebApp/library/source/pages/account_page/risk_page/risk_controller.tsx

```diff
@@ -56,12 +55,12 @@ export class RiskController extends React.Component<Properties, State> {
       () => {
         this.setState({
           isLoaded: true
+        });
```

Comment:
Rewrite ```componentDidMount``` to be async and do that for any pages you come across in the future.

    public async componentDidMount(): void {
      await this.props.model.load();
      this.setState({
        isLoaded: true
      });
    }